src/library/basesqltablemodel.cpp

```diff
@@ -723,6 +723,10 @@ QVariant BaseSqlTableModel::data(const QModelIndex& index, int role) const {
     // role
     switch (role) {
         case Qt::ToolTipRole:
+            if (column == fieldIndex(ColumnCache::COLUMN_LIBRARYTABLE_COLOR)) {
```

Comment:
Wouldn't QVariant() work be more appropriate here?

Reply:
This works for me:

```
const auto color = mixxx::RgbColor::optional(value);
if (color) {
    value = mixxx::toQColor(color).name();
} else {
    value = QString();
}
```

Reply:
Maybe add a reusable, free function to `rgbcolor.h`:
`QString nullableName(mixxx::RgbColor::optional_t color);`

Please note that QColor().name() seems to return "#000000", i.e. black although undefined/invalid. So we should test this new function!

Not sure about the naming, but at least it is concise and easy to remember.

Reply:
Do you want me to do this as part of this PR?

Reply:
Hold on. Let's first merge this PR and I will publish a separate PR then.